roles/confluent.control_center/molecule/kerberos/roles/kerberos_server/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
```

Comment:
file seems unnecessary

Reply:
It's for ansible galaxy support.  Was added originally as we thought we may support galaxy in the future.